MyEnumerable.rb

```diff
@@ -0,0 +1,23 @@
+module MyEnumerable
+
+   def all?
+      each { |e| return true if yield (e) }
+      false
+   end
+
+   def any?
+      each { |e| return true if yield (e) }
+      false
+   end
+
+   def select
+      arr = []
+      each { |e|
+      if e%2==0
+         arr.push(e)
+      end
+      }
+      return arr
+   end
+
```

Comment:
- [ ] Instead of select the requirement was to create the filter method 
![image](https://github.com/David-Lanzz/MyEnumerable/assets/62705947/11ef6306-c0fb-46f9-a928-ee2fdcfc99c8)